pkg/controller/admissionchecks/multikueue/multikueuecluster.go

```diff
@@ -102,30 +125,66 @@ func (rc *remoteClient) setConfig(watchCtx context.Context, kubeconfig []byte) e
 	}
 	rc.client = remoteClient
 
-	newWatcher, err := rc.client.Watch(watchCtx, &kueue.WorkloadList{})
+	// TODO: Filter based on multikueue-origin label when available.
```

Comment:
Remove the TODO since the code already is using the label

Reply:
done